nunit-console.sls

```diff
@@ -1,9 +1,16 @@
+{% set versions = {'3.9':['0']} %}
```

Comment:
You need to find a way to support older versions of this software. You may need to create two loops. One that handles the `v` that is used in 3.9 and one that handles the rest where it's just the version without the `v`

Reply:
I've added an extra loop for the path